bindings/meson.build

```diff
@@ -0,0 +1,9 @@
+build_rust = not get_option('rs').disabled()
+if build_rust
+	if not build_gir
+		error('cannot build "rust" without "introspection"')
```

Comment:
rs auto + gir auto/disabled = error ?

Should probably find programs `cargo` and `generator.py` here, have `build_rust` set depending on their found() values and error when enabled only. Please see how this is done for `vapi` in project.

Reply:
should be fixed. I naively assumed that meson "features" just behave like booleans with fancy name